csharp/ql/integration-tests/windows-only/standalone_dependencies/Assemblies.expected

```diff
@@ -163,54 +161,7 @@
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/System.Xml.XPath.dll |
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/System.Xml.XmlDocument.dll |
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/System.Xml.XmlSerializer.dll |
+| /microsoft.netcore.app.ref/7.0.2/ref/net7.0/WindowsBase.dll |
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/mscorlib.dll |
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/netstandard.dll |
-| /microsoft.windowsdesktop.app.ref/7.0.2/ref/net7.0/Accessibility.dll |
```

Comment:
I think we need to keep these `microsoft.windowsdesktop.app.ref` DLLs. (Or we would need to find a way to recognize projects that need them, similarly to what we do with asp.net core now. But it seems easier to just keep them now.)

Reply:
Then I will introduce some logic that always uses this nuget package if it is downloaded.